### PR TITLE
Run tests using testem

### PR DESCRIPTION
**Pull request by marcioj**

### CODE REVIEW

File: Brocfile.js [hunk 1]
```diff
@@ -74,4 +77,46 @@ const bundle = concat(mergeTrees([vendor, main]), {
...
+    moduleIds: true,
+    modules: 'amd',
+    sourceMaps: true,
+  });
+
+  test = mergeTrees([bundle, testVendor, test, testIndex]);
+
+  test = testem(test, {
+    ci: process.env.CI === 'true',
+    "framework": "mocha+chai",
```
Comment (jayphelps) [inline Brocfile.js]: Single quotes please :dancer: (several lines)

File: dist/graffiti-runtime.js [hunk 2]
```diff
@@ -5933,35 +5933,35 @@ define("dom-helper", ["exports", "module", "./htmlbars-runtime/morph", "./morph-
```
Comment (jayphelps) [inline dist/graffiti-runtime.js]: Don't include a new copy of dist/graffiti-runtime.js please

**Comment by jayphelps** [comment]
@marcioj ping me when those changes are ready :+1:

**Comment by marcioj** [comment]
@jayphelps done

**Pull request closed without merging**